cypress/integration/executions_spec.js

```diff
@@ -37,8 +37,7 @@ describe('Dashboard Executions Page', () => {
         .next().contains('li', '1 Running');
 
       // shows a list of executions with IDs and status
-      const executionStatusFile = './test/fake-api-fixtures/executions/index.json';
-      cy.readFile(executionStatusFile).as('executionStatus');
+      cy.getFakeApiFixture('executions').as('executionStatus');
```

Comment:
Completely unnecessary change but I wanted to centralize how we load test data from fake API fixtures so we don't have to update the path in multiple places when necessary